source/contracts/reporting/Market.sol

```diff
@@ -145,7 +146,11 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
 
     function contribute(uint256[] _payoutNumerators, bool _invalid, uint256 _amount) public returns (bool) {
         require(getInitialReporter().getReportTimestamp() != 0);
-        require(!feeWindow.isOver());
+        if (disputePacingOn) {
+            require(feeWindow.isActive());
+        } else {
+            require(!feeWindow.isOver());
+        }
```

Comment:
If we have triggered disputePacing for this market, require that the feeWindow is currently active. successfully disputing pushes a market to the next fee window, ensuring this require will fail until that feeWindow's time has come.

If not dispute pacing, just make sure we haven't expired the current window